hammerblade/torch/kernel/kernel_threshold.cpp

```diff
@@ -30,6 +31,7 @@ extern "C" {
         } else {
           return other;
         }
+        printf("self and other are : \n", self, other);
```

Comment:
cant have printf in the kernels -- cant build cosim ;)

Reply:
Done.